### PR TITLE
Added route and view for Join Our Team.

### PR DESCRIPTION
**Pull request by jainarunarunjain**
@deshraj @spyshiv @aka-jain Please Review the PR.

### CODE REVIEW

File: frontend/src/js/route-config/route-config.js [hunk 1]
```diff
@@ -309,13 +309,21 @@
...
         var our_team = {
             name: 'our-team',
             url: "/team",
             templateUrl: baseUrl + "/web/our-team.html",
             title: "Our Team"
         };
 
+        var join_our_team = {
+            name: 'join-our-team',
+            url: "/join",
```
Comment (aka-jain) [inline frontend/src/js/route-config/route-config.js]: @jainarunarunjain make url "join-us"

Comment (jainarunarunjain) [inline frontend/src/js/route-config/route-config.js]: @aka-jain Sure.

**Comment by jainarunarunjain** [comment]
@deshraj @aka-jain @spyshiv Please have a look.

**Comment by aka-jain** [comment]
Thanks @jainarunarunjain. Good for now.

**Pull request merged**